crates/ruff/src/rules/pyupgrade/rules/use_pep695_type_alias.rs

```diff
@@ -1,4 +1,9 @@
-use ruff_python_ast::{Expr, ExprName, Ranged, Stmt, StmtAnnAssign, StmtTypeAlias};
+use ruff_python_ast::{
```

Comment:
We often do `use ruff_python_ast::{self as ast, ...}` for convenience / by convention. Then you can do `ast::TypeParams` instead of `ruff_python_ast::TypeParams` below.